solidity/contracts/PhasedEscrow.sol

```diff
@@ -66,8 +66,96 @@ contract CurveRewardsEscrowBeneficiary is Ownable {
         curveRewards = _curveRewards;
     }
 
-    function __escrowSentTokens(uint256 amount) external {
+    function __escrowSentTokens(uint256 amount) external onlyOwner {
```

Comment:
Given that this `onlyOwner` turns to be an important requirement, we should cover it with unit tests inside `TestPhasedEscrow.js`. Ideally, we should do the same for `StakerRewardsBeneficiary` as well:
```
describe("CurveRewardsEscrowBeneficiary", () => {
  describe("__escrowSentTokens", async () => {
    it("can be called by owner", async () => {

    })

    it("can not be called by non-owner", async () => {
    })    
  })
})
```

Reply:
Added.